bin/gendocs

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash -e
 
+# Automatic document generation, validation and helpers
+#
+# bin/gendocs ...
+# 
+# <unspecified> (default)
+#   Generate HTML documentation from JSON schema. Stored in gencode/docs
+#
+# check
+#   Compares the hashes from the last time gendocs was run 
+#
+# feature_bucket [input_file] [output_file]
+#   Takes a tab delimited text file containing the list of feeature buckets
```

Comment:
oh no, not tabs, please. They are horrible. Can this be CSV (comma)... much more sane. Should be consumable from an "export CSV" from a sheet...